docs/framework/20sources/30implementation/pause.md

```diff
@@ -1,14 +1,45 @@
 ### Ability to Pause Project
-Smart contracts have functions that execute parts of the project. 
-For example, a function could mint coins based on actions on 
-another blockchain. The ability to “pause” a function in a 
+All Data Plane functions should be *pausable*. For example, a bridge contract could 
+have a function that could transfer coins based on actions on another blockchain. 
+The ability to *pause* a function in a 
 project allows administrators to stop functions from successfully 
 executing. If there is a vulnerability that is being actively 
 exploited in a project, having the ability to pause a function 
-could stop the exploitation of the project mid way through the 
+could stop the exploitation of the project midway through the 
 attack.
 
-When analyzing whether a project can be paused, it is important 
-to check whether all data processing functions can be paused, or 
-just some parts of the project. The more of data processing 
-functions that can be paused, the less risky the project is.
+For Ethereum based projects, the OpenZeppelin project has an example 
+ contract [Pausable.sol](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/security/Pausable.sol)
+that can be used to implement pausing. Using this template, 
+pausing a function becomes as simple as adding a modifier **whenNotPaused**. 
```

Comment:
nit: suggest we use in-line code markers (i.e. backticks ` ) when referring to functions as opposed to bold/italics.

Reply:
done